google_symptoms/delphi_google_symptoms/pull.py

```diff
@@ -95,45 +95,6 @@ def preprocess(df, level):
     return df
 
 
-def get_date_range(export_start_date, export_end_date, num_export_days):
-    """Produce date range to retrieve data for.
-
-    Calculate start of date range as a static offset from the end date.
-    Pad date range by an additional 7 days before the earliest date to
-    produce data for calculating smoothed estimates.
-
-    Parameters
-    ----------
-    export_start_date: date
-        first date to retrieve data for
-    export_end_date: date
-        last date to retrieve data for
-    num_export_days: int
-        number of days before end date to export
-
-    Returns
-    -------
-    list
-    """
-    PAD_DAYS = 7
-
-    if num_export_days == "all":
```

Comment:
I wasn't sure; if you would want to have something similar, it would be better to have a seperate parameter like backfill_all as a boolean personally, and have the num_export_days to exclusive be a postitve integer for clarity, but also didn't want to remove remove it so it's at an akward stage

Reply:
The "all" setting isn't used in production, it's only meant as a convenience for local testing, so it's not super important to support. If the new version already supports it, I'd say leave as-is; if the new version doesn't support it, I wouldn't bother adding in your suggested new `backfill_all` param.